dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/util/ContentletUtil.java

```diff
@@ -44,6 +45,11 @@ public static boolean isFieldTypeAllowedOnImportExport(final Field field){
 		return !fieldTypesToExcludeFromImportExport.contains(field.getFieldType());
 	}
 
+	public static boolean isHost(final Contentlet contentlet){
+		final Structure hostStrucuture = CacheLocator.getContentTypeCache().getStructureByVelocityVarName("Host");

```

Comment:
Host.HOST_VELOCITY_VAR_NAME there is a constant for Host

Reply:
done.